docs/plugins/musicbrainz.rst

```diff
@@ -131,10 +132,11 @@ Default
     available the release ID will be extracted from the URL provided and imported to
     the beets library.
 
-    The library fields of the corresponding :ref:`autotagger_extensions` are used to
-    save the data as flexible attributes (``discogs_album_id``, ``bandcamp_album_id``, ``spotify_album_id``,
-    ``beatport_album_id``, ``deezer_album_id``, ``tidal_album_id``). On re-imports
-    existing data will be overwritten.
+.. conf:: _genres_tag
```

Comment:
Looks like another rebase or merge of master is required. You didn't want to remove the lines above this one I guess. Might have been a change on master recently?

Reply:
Oops, not sure how I did that.  I rebased instead again and squashed for good measure.

Reply:
looks good now. @snejus you want to take a final look and approve it?